### ISSUE TITLE
ZSTD update for possible better performance

### ISSUE THREAD

**Issue by stormi** [xcp-ng/xcp#299]
XCP-ng allows to use ZSTD to compress VM exports.

I'm not sure I'll push that update to everyone, but for those willing to test it, just update the `zstd` package:

```
yum update zstd --enablerepo=xcp-ng-testing
```

The changelog mentions the following performance improvements: 
* decompression speed faster by ~7% at the default compression level (that we use)

**Comment by stormi** [comment]
Updated to 1.4.2 in XCP-ng 8.1. Sources for 1.4.4 are available too but not present in EPEL for now, so to upgrade to that I'd need to switch to building it ourselves rather than importing the package from EPEL.

1.4.4 is supposed to bring a 10% decompression speed boost, but given the maintenance overhead of building it ourselves I'll not update until we've got someone (could be me, later on) available to do benchmarks (with a package I would provide) that show a significant gain in the export/import use case.
